core/sr-primitives/src/traits.rs

```diff
@@ -1147,9 +1167,12 @@ impl Printable for &str {
 	}
 }
 
-impl Printable for u64 {
+#[impl_for_tuples(1, 12)]
+impl Printable for Tuple {
+	for_tuples!( where #( Tuple: Printable )* );
```

Comment:
```suggestion
```

This is done automatically :) 

The syntax is only required when you need another bound.